low-level/sspl-ll.spec

```diff
@@ -76,9 +76,11 @@ cp -p $SSPL_SETUP/consuldump.py $SSPL_BASE/low-level/
 
 %pre
 # take backup of cache folder if exists
-mkdir -p /opt/seagate/backup/%{version}
-[ -f /etc/sspl.conf ] && cp -p /etc/sspl.conf /opt/seagate/backup/%{version}/sspl.conf
-[ -d /var/%{product_family}/sspl ] && cp -Rp /var/%{product_family}/sspl /opt/seagate/backup/%{version}/
+mkdir -p /opt/seagate/%{product_family}/backup/%{version}/sspl
+[ -f /etc/sspl.conf ] && cp -p /etc/sspl.conf /opt/seagate/%{product_family}/backup/%{version}/sspl/sspl.conf
+if [ -d /var/%{product_family}/sspl/data ]; then
+    cp -Rp /var/%{product_family}/sspl/data /opt/seagate/%{product_family}/backup/%{version}/sspl/
+fi
```

Comment:
Why this is failing now, the same code used to work earlier

Reply:
Earlier we had directory creation after this condition. rc was still same at this condition but after the directory creation execution rc was 0.
So we hadn't faced this issue.